src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/xlf/LocalizableStrings.ja.xlf

```diff
@@ -30,7 +30,7 @@
       </trans-unit>
       <trans-unit id="Authoring_ManualInstructionIdIsNotUnique">
         <source>'id' of the manual instruction '{0}' at index {1} is not unique. Only the first manual instruction that uses this id will be localized.</source>
-        <target state="translated">インデックス {1} の手動手順 \"{0}\" の 'ID' が一意ではありません。この ID を使用する最初の手動手順のみがローカライズされます。</target>
+        <target state="translated">インデックス {1} の手動手順 {0} の 'ID' が一意ではありません。この ID を使用する最初の手動手順のみがローカライズされます。</target>
```

Comment:
Single quotes are missing around the format item (`{0}`)

Reply:
Doesn't meet the priority bar for fixing (as it's just a formatting issue, not an acutal content or translation issue) - so closing